main/src/ca/uwaterloo/flix/language/ast/SyntaxTree.scala

```diff
@@ -226,6 +226,8 @@ object SyntaxTree {
 
       case object FixpointSelect extends Expr
 
+      case object FixpointPSolve extends Expr
```

Comment:
I am thinking we should perhaps call this `FixpointPSolveWithProject` -- for consistency

Reply:
Could there not be? Perhaps its unclear what the semantics would be then?

Reply:
If there is no project, then I suggest we call it `FixpointSolveWithProvenance`